odoo-shippable/scripts/library.sh

```diff
@@ -58,3 +58,20 @@ createuser_custom(){
 psql_create_role(){
     su - postgres -c "psql -c  \"CREATE ROLE ${1} LOGIN PASSWORD '${2}' SUPERUSER INHERIT CREATEDB CREATEROLE;\""
 }
+
+service_postgres_without_sudo(){
+    USER="${1}"
+    VERSIONS=$(pg_lsclusters  | sed '1d' | awk '{print $1}' )
+    for version in $VERSIONS; do
+        pg_dropcluster --stop $version main
+    done
+    adduser ${USER} postgres
+    chown -R ${USER}:${USER} /var/run/postgresql
+    for version in $VERSIONS; do
+        pg_createcluster -u ${USER} -g postgres -s /var/run/postgresql --start-conf auto --start $version main
```

Comment:
@mapuerta he wrote **1**5432, besides I think it is better to stop the cluster before starting the new one, I guess we won't have more than one postgres version running at the same time if it is trying to read the port still put any number there (a wired one to do it faster), because it will be replaced [here](https://github.com/Vauxoo/docker-odoo-image/blob/master/odoo-shippable/scripts/build-image.sh#L345) later